auth/credentials/clientcredentials.go

```diff
@@ -0,0 +1,78 @@
+package credentials
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"golang.org/x/oauth2"
+	"golang.org/x/oauth2/clientcredentials"
+	"io"
+	"net/http"
+	"net/url"
+	"strings"
+)
+
+// Config describes a 2-legged OAuth2 flow, with both the
+// client application information and the server's endpoint URLs.
+type Config struct {
+	clientcredentials.Config
+	RevokeURL string
+	userAgent string
+}
+
+// tokenSource manages the OAuth Token fetching and refreshing using a LocalTokenCache.
```

Comment:
`LocalTokenCache` seems to have been removed?

Reply:
correct